client/gutenberg/editor/calypsoify-iframe.tsx

```diff
@@ -490,6 +491,7 @@ const mapStateToProps = ( state, { postId, postType, duplicatePostId }: Props )
 		'frame-nonce': getSiteOption( state, siteId, siteOption ) || '',
 		'jetpack-copy': duplicatePostId,
 		origin: window.location.origin,
+		environment_id: config( 'env_id' ),
```

Comment:
Non-blocking but JS normally uses camelCase vs snake_case

Reply:
Is this one of those rare cases where JS caters to PHP, rather than the other way around?

Reply:
I've updated to `environment-id ` (with `-`) as that follows the patterns established with `frame-nonce` .etc.